pkg/cmd/attestation/download/download.go

```diff
@@ -122,7 +122,7 @@ func runDownload(opts *Options) error {
 
 	opts.Logger.VerbosePrintf("Downloading trusted metadata for artifact %s\n\n", opts.ArtifactPath)
 
-	c := verification.FetchAttestationsConfig{
+	c := verification.FetchRemoteAttestations{
```

Comment:
could this be… a Params object?

Reply:
as in 

```suggestion
	c := verification.FetchRemoteAttestationsParams{
```

and does that clarify that we could make the APIClient a separate argument in the `GetRemoteAttestations` func? as in `GetRemoteAttestations(client, params)`?